components/post_view/post_list.jsx

```diff
@@ -260,7 +262,7 @@ export default class PostList extends React.PureComponent {
             }
         } else {
             this.loadingPosts = false;
-            if (this.mounted) {
+            if (this.mounted && this.props.posts) {
```

Comment:
the selector to get the posts can return null so the next line was causing an error